### PR TITLE
TE-240 Slideshow widget

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-240)

### What **one** thing does this PR do?
Creates component, tests and examples for the Slideshow widget

### CODE REVIEW

File: src/components/widgets/Slideshow/component.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImageGallery from 'react-image-gallery';
+import 'react-image-gallery/styles/css/image-gallery-no-icon.css';
```
Comment (davidpelayo) [inline src/components/widgets/Slideshow/component.js]: Maybe style imports below JS imports?

File: src/components/widgets/Slideshow/utils/adaptImages.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ * @return {Object[]}
+ */
+export const adaptImages = images =>
+  images.map(({ alternativeText, url, sizes, sourceSet, title }) => ({
+    original: url,
+    originalAlt: alternativeText,
+    originalTitle: title,
+    sizes,
+    srcSet: sourceSet,
+  }));
```
Comment (davidpelayo) [inline src/components/widgets/Slideshow/utils/adaptImages.js]: Nice!

File: src/semantic/src/themes/livingstone/collections/menu.overrides [hunk 3]
```diff
@@ -175,6 +175,56 @@
...
+        max-height: @slideshowImageMaxHeight;
+      }
+    }
+
+    // This `span` wraps the left and right nav buttons.
+    > span:first-child {
+      display: flex;
+      justify-content: space-between;
+      padding: 0 @slideshowButtonHorizontalPadding;
+      position: absolute;
```
Comment (davidpelayo) [inline src/semantic/src/themes/livingstone/collections/menu.overrides]: Ensure the context maybe by setting `position:relative` to the parent?

File: src/semantic/src/themes/livingstone/collections/menu.overrides [hunk 4]
```diff
@@ -175,6 +175,56 @@
...
+
+    > .image-gallery-bullets  {
+      bottom: @slideshowBulletsBottomPosition;
+
+      > .image-gallery-bullets-container button.image-gallery-bullet {
+        background-color: @paginationLabelBackground;
+        border: none;
+        box-shadow: none;
+        margin: 0;
+        margin-left: @paginationLabelHorizontalMargin;
```
Comment (davidpelayo) [inline src/semantic/src/themes/livingstone/collections/menu.overrides]: why not using directly the shorthand expression?

File: src/semantic/src/themes/livingstone/collections/menu.overrides (hunk 3 above)
Comment (marcdavi-es) [reply src/semantic/src/themes/livingstone/collections/menu.overrides]: It already has it from the library :)

**Pull request merged**